src/main/java/org/traccar/database/DeviceManager.java

```diff
@@ -390,7 +390,7 @@ private Object lookupAttribute(long deviceId, String attributeName, boolean look
             if (result == null) {
                 if (lookupConfig) {
                     result = Context.getConfig().getString(attributeName);
-                } else {
+                } else if (lookupServer) {
```

Comment:
I think it would make sense to remove `else`. In future we might potentially want to use both.

Reply:
I thought the same, but as I said in my first comment, just wanted to keep same behavior. No problem, removing it. Thanks.

Reply:
It's not a public method, so we don't have to keep existing behavior for this one.

Reply:
You're right. Working on that...